lib/index.js

```diff
@@ -36,20 +41,88 @@ class AttributeList {
 
     this.el.setAttribute(this.attributeName, this.tokenList.join(" "))
   }
-}
 
-export function $al(el) {
-  function toCamelCase(str){
-    return str.replace(/^([A-Z])|[\s-_]+(\w)/g, (match, p1, p2, offset) =>  p2 ? p2.toUpperCase() : p1.toLowerCase());
+  remove(attr){
+    var parsed = AttributeList.parseAttributeFakeTokenList(arguments, 'add');
+    parsed.forEach(token => {
+      let pos = this.tokenList.indexOf(token)
+      if (pos > -1){
+        this.tokenList.splice(pos, 1)
+      }
+    })
+
+    this.el.setAttribute(this.attributeName, this.tokenList.join(" "))
+  }
+
+  toggle(attr){
+    let pos = this.tokenList.indexOf(attr)
+    if (pos > -1){
+      this.remove(attr)
+    } else {
+      this.add(attr)
+    }
+    this.el.setAttribute(this.attributeName, this.tokenList.join(" "))
   }
+}
+
+
+// Helper
+function toCamelCase(str){
+  return str.replace(/^([A-Z])|[\s-_]+(\w)/g, (match, p1, p2, offset) =>  p2 ? p2.toUpperCase() : p1.toLowerCase());
+}
+
+
+/*
+ * @name $attributeList
+ * @description AttributeList Wrapper Mode, se like _loadash or jquery
+ *
+ * @param {HTMLElement|String} el — target element
```

Comment:
explains that returns an object with all attributes on an element, camel cased and a corresponding AttributeList instance (with all the method goodies )